### ISSUE TITLE
Mac Version?

### ISSUE THREAD

**Issue by PlasmStudios** [finndev/PokeBuddy#374]
Is there a possibility for a Mac Version soon?

**Comment by curtiola** [comment]
Couldn't tell ya, but you can just emulate windows for now, Google is your friend

**Comment by PlasmStudios** [comment]
Haha, thanks. I have Windows running via Bootcamp. It's just a pain switching back and forth Operating Systems.

**Comment by arjunvegda** [comment]
@PlasmStudios You can use VirtualBox.

**Comment by futi666** [comment]
throw ur mac into a trashcan and then buy a windows laptop/pc - problem solved

**Comment by lordofthereef** [comment]
I run on a mac via bootcamp as well. I guess it helps that I prefer windows as an OS, namely because of games, so there's that. Are you doing something frequently in OSX that you can't just do in windows?

**Comment by PlasmStudios** [comment]
@lordofthereef Nah, I'm just a lazy ass. I'm a designer so Mac OS is my daily and preferred driver. I just find it to be a inconvenience to switch back and forth from OS to OS (restarts and all). Either ways, I ran PokeBuddy for about 2 hours today so I think good for the day. I'm also considering moving entirely to a Windows laptop; Looking at a Dell XPS15 9550 as a option.

**Comment by curtiola** [comment]
Does VirtualBox require a restart? I don't know anything about mac except I don't want one haha but arjunvegda suggested it, could be a cheaper solution for you
